code/modules/mob/living/basic/blob_minions/blob_spore.dm

```diff
@@ -0,0 +1,127 @@
+	health = BLOBMOB_SPORE_HEALTH
+	maxHealth = BLOBMOB_SPORE_HEALTH
+	bubble_icon = "blob"
+	speak_emote = null
+	verb_say = "psychically pulses"
+	verb_ask = "psychically probes"
+	verb_exclaim = "psychically yells"
+	verb_yell = "psychically screams"
+	melee_damage_lower = BLOBMOB_SPORE_DMG_LOWER
+	melee_damage_upper = BLOBMOB_SPORE_DMG_UPPER
+	habitable_atmos = list("min_oxy" = 0, "max_oxy" = 0, "min_plas" = 0, "max_plas" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	minimum_survivable_temperature = 0
+	maximum_survivable_temperature = INFINITY
+	obj_damage = 0
+	attack_verb_continuous = "batters"
+	attack_verb_simple = "batter"
+	attack_sound = 'sound/weapons/genhit1.ogg'
+	death_message = "explodes into a cloud of gas!"
+	lighting_cutoff_red = 20
+	lighting_cutoff_green = 40
+	lighting_cutoff_blue = 30
+	initial_language_holder = /datum/language_holder/empty
+	gold_core_spawnable = HOSTILE_SPAWN
+	basic_mob_flags = DEL_ON_DEATH
+	ai_controller = /datum/ai_controller/basic_controller/blob_spore
+	/// Size of cloud produced from a dying spore
+	var/death_cloud_size = 1
+	/// Type of mob to create
+	var/mob/living/zombie_type = /mob/living/basic/blob_zombie
+
+/mob/living/basic/blob_spore/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/simple_flying)
+	add_traits(list(TRAIT_BLOB_ALLY, TRAIT_MUTE), INNATE_TRAIT)
+	AddElement(/datum/element/swabable, CELL_LINE_TABLE_BLOBSPORE, CELL_VIRUS_TABLE_GENERIC_MOB, 1, 5)
+	AddComponent(/datum/component/blob_minion, on_strain_changed = CALLBACK(src, PROC_REF(on_strain_updated)))
+
+/mob/living/basic/blob_spore/death(gibbed)
+	. = ..()
+	death_burst()
+
+/// Create an explosion of spores on death
+/mob/living/basic/blob_spore/proc/death_burst()
+	do_chem_smoke(range = death_cloud_size, holder = src, location = get_turf(src), reagent_type = /datum/reagent/toxin/spore)
+
+/// Doesn't actually do anything if you're not spawned by a blob
+/mob/living/basic/blob_spore/proc/on_strain_updated(mob/camera/blob/overmind, datum/blobstrain/new_strain)
+	return
+
+/mob/living/basic/blob_spore/melee_attack(mob/living/carbon/human/target, list/modifiers, ignore_cooldown)
+	. = ..()
+	if (!ishuman(target) || target.stat != DEAD)
+		return
+	zombify(target)
+
+/// Become a zombie
+/mob/living/basic/blob_spore/proc/zombify(mob/living/carbon/human/target)
+	visible_message(span_warning("The corpse of [target.name] suddenly rises!"))
+	var/mob/living/basic/blob_zombie/blombie = change_mob_type(zombie_type, loc, new_name = initial(zombie_type.name))
+	blombie.set_name()
+	if (istype(blombie)) // In case of badmin
+		blombie.consume_corpse(target)
+	SEND_SIGNAL(src, COMSIG_BLOB_ZOMBIFIED, blombie)
+	qdel(src)
+
+/// Variant of the blob spore which is actually spawned by blob factories
+/mob/living/basic/blob_spore/minion
+	gold_core_spawnable = NO_SPAWN
+	zombie_type = /mob/living/basic/blob_zombie/controlled
+	/// We die if we leave the same turf as this z level
+	var/turf/z_turf
```

Comment:
```suggestion
	/// We die if we leave the same z as the z this turf is on
	var/turf/z_turf
```
Also I find it kind of goofy that we're tracking the turf instead of just the Z we want, ie `5`. Though I understand why, for multi-z station support.